platform/api/esx/api.go

```diff
@@ -282,6 +282,32 @@ func (a *API) GetConsoleOutput(name string) (string, error) {
 	return string(buf), nil
 }
 
+func (a *API) CleanupFiles(name string) error {
```

Comment:
The method name is correct from an implementation perspective, but from the perspective of the rest of mantle, its place in the machine lifecycle is a bit unclear: CreateDevice -> TerminateDevice -> CleanupFiles.  What about calling it CleanupDevice?

Is it worth adding a check that the device has already been terminated?